common4j/src/main/com/microsoft/identity/common/java/commands/parameters/BrokerInteractiveTokenCommandParameters.java

```diff
@@ -77,6 +77,9 @@ public class BrokerInteractiveTokenCommandParameters extends InteractiveTokenCom
     @Expose
     private final String homeTenantId;
 
+    // Parameter representing if this broker request is an ATv2 request
```

Comment:
nit: use full name (Account Transfer instead of AT. AT could also mean Access Token)

Reply:
Sounds good